### PR TITLE
Onboarding in ens:// spec

### PR DESCRIPTION
**Pull request by mikehelmick**
## Proposed Changes

* define ens://onboard in the ens:// spec
* no implementation yet.
* 
**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: docs/ens-spec.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ to share within the exposure notifications system.
 
 -   `ens` : describes that the exposure notifications application to be opened.
 
+-   `onboard` : activate exposure notifications. 
+
 -   `v` : verify, currently this is the only available action.
```
Comment (sethvargo) [inline docs/ens-spec.md:19]: I think this comment is wrong now?

File: docs/ens-spec.md [hunk 2]
```diff
@@ -55,3 +63,17 @@ And for State of New South Wales in Austraila:
...
+```text
+ens://onboard
+```
+
+### Activation, region in URI
+
+For the State of Washington, in the United States:
+
+```text
+ens://onboard?r=WA-WA
```
Comment (sethvargo) [inline docs/ens-spec.md:78]: US-WA?

Comment (sethvargo) [inline docs/ens-spec.md:78]: Add a note on case sensitivity?

**Review by sethvargo — CHANGES_REQUESTED** [review]

File: docs/ens-spec.md (hunk 2 above)
Comment (mikehelmick) [reply docs/ens-spec.md:78]: added note above on region param

**Review by sethvargo — APPROVED** [review]
/lgtm

**Pull request merged**